var/spack/repos/builtin/packages/py-pygments/package.py

```diff
@@ -23,6 +23,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+import os
```

Comment:
@mathstuf This must have been the problem package, but I've just removed this line and reinstalled without issues. So I'll revert the change. I have no idea as to what the problem could have been...